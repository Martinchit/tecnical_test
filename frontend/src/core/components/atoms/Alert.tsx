import styled from 'styled-components';
import { Alert as A } from 'react-bootstrap';

export const Alert = styled(A)`
  width: 100%;
  margin: 0;
`;
