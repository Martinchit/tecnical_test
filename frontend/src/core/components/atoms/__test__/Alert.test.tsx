import React from 'react';
import { shallow } from 'enzyme';

import { Alert } from '../Alert';

describe('Alert', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Alert />);
    expect(wrapper).toMatchSnapshot();
  });
});
