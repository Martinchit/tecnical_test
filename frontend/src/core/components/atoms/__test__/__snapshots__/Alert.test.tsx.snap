// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert renders correctly 1`] = `
<Alert
  className="sc-AxjAm hCTAab"
  closeLabel="Close alert"
  show={true}
  transition={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "appear": false,
        "in": false,
        "mountOnEnter": false,
        "timeout": 300,
        "unmountOnExit": false,
      },
      "displayName": "Fade",
      "render": [Function],
    }
  }
/>
`;
